http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java

```diff
@@ -202,7 +208,7 @@ protected void doTestConnectTimeout(boolean proxyTest)
         }
     }
 
-    @Test(expectedExceptions = ConnectException.class)
+    @Test
     public void testConnectionRefused()
             throws Exception
```

Comment:
remove?

Reply:
@ssheikin I'm aware and will cleanup as a follow up

Reply:
Added